src/fix/main.cpp

```diff
@@ -1041,13 +1043,12 @@ static void processFile(int input, int output, char const *name, off_t fileSize)
 			assert(rom0Len == sizeof(rom0));
 		}
 		assert(nbBanks >= 2);
+		// The subtraction would be invalid for 0, but the assertion above means it won't happen.
+		auto logNbBanks = std::bit_width(nbBanks - 1);
```

Comment:
It definitely returns an [`int`](https://en.cppreference.com/w/cpp/numeric/bit_width).
```suggestion
		int logNbBanks = std::bit_width(nbBanks - 1);
```